docs/how-to-run.adoc

```diff
@@ -53,6 +53,13 @@ This script requires Omicron be uninstalled, e.g., with `pfexec
 that is not the case. The script will then remove the file-based vdevs and the
 VNICs created by `create_virtual_hardware.sh`.
 
+=== Make me a certificate!
+
+Nexus's external interface will typically be served using public-facing x.509
+certificate. While we are still configuring the mechanism to integrate this real
+certificate into the package system, `./tools/create_self_signed_cert.sh` can be
+used to generate an equivalent self-signed certificate.
```

Comment:
TODO(me): I wonder if I can fix https://github.com/oxidecomputer/omicron/issues/1398 and make this easier at  the same time?

Reply:
See: https://github.com/oxidecomputer/omicron/issues/1528. We're gonna need to grab / insert certs from *somewhere*. 

I've updating the packaging hints to make this more obvious if an error is encountered.